docs/get-started.md

```diff
@@ -1,7 +1,14 @@
 # Get Started
 
 To get started with Qlik Core and Qlik Associative Engine, and to learn the basic concepts,
-we provide a number of examples and tutorials.
+we provide a number of tools, examples and tutorials.
+
+You can explore and play around with the
+[core-experimenter](https://experimenter.netlify.com/), a tool which showcaseses the Qlik Associative Engine's selection
```

Comment:
showcaseses -> showcases

Reply:
Maybe we want to use https://experimenter.qlik.dev/ instead?